povm_toolbox/quantum_info/multi_qubit_frame.py

```diff
@@ -45,17 +50,22 @@ class MultiQubitFrame(BaseFrame[int]):
        for more general information.
     """
 
-    def __init__(self, list_operators: list[Operator]) -> None:
+    def __init__(
+        self, list_operators: list[Operator], shape: tuple[int, ...] | None = None
```

Comment:
```suggestion
        self, list_operators: list[Operator], *, shape: tuple[int, ...] | None = None
```
Let's make `shape` a keyword-only argument